woocommerce/woocommerce
# Change "District Of Columbia" to "District of Columbia"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out a product on any WooCommerce store.
2. Following the selection of the United States as the country, navigate to the state field.
3. Check if the label for DC is "District Of Columbia" or "District of Columbia."



### Changelog entry


This pull request fixes a minor typo.
<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Change "District Of Columbia" to "District of Columbia".

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
